tests/JsonSchema/Tests/Drafts/Draft4Test.php

```diff
@@ -12,18 +25,15 @@ protected function getFilePaths()
         );
     }
 
+    /**
+     * {@inheritdoc}
+     */
     protected function getSkippedTests()
     {
         return array(
-            // Not Yet Implemented
-            'definitions.json',
-            // Partially Implemented
-            'ref.json',
-            'refRemote.json',
```

Comment:
BOOM! :boom: 

Really stoked about this expansion of features for this package.